src/context-menus/message/close-poll.js

```diff
@@ -0,0 +1,34 @@
+const Discord = require('discord.js');
+const { ContextMenuCommandBuilder } = require('@discordjs/builders');
+const { ApplicationCommandType } = require('discord-api-types/v10');
+const pollUtils = require('../../utils/polls');
+
+
+/**
+ *
+ * @param {Discord.ContextMenuInteraction} interaction
+ */
+async function closePollHandler(interaction) {
+	const { targetId } = interaction;
+
+	const message = await interaction.channel.messages.fetch(targetId);
+
+	pollUtils.closePoll(message);
+
+	await interaction.reply({
+		content: 'Poll closed!',
+		ephemeral: true
+	});
+}
+
+const contextMenu = new ContextMenuCommandBuilder();
+
+contextMenu.setDefaultMemberPermissions(Discord.PermissionFlagsBits.ManageMessages);
```

Comment:
This is meant to be insecure so that moderators can create and close polls. It looks like setting the actual command to manage messages slipped my mind though. >~<

Reply:
Lowered security on `/poll` to match.